manifests/plugin/rabbitmq.pp

```diff
@@ -97,6 +97,15 @@
     install_options => $install_options,
   }
 
+  $rt = '/usr/local/share/collectd-rabbitmq/types.db.custom' # RabbitMQ addon types file
+  $ct = '/usr/share/collectd/types.db'                       # CollectD types file
+
+  exec { 'update_typesdb':
+    command => "/bin/cat ${rt} >> ${ct}",
+    unless  => "[ `cat ${ct} | sort -u | wc -l` -eq `cat ${rt} ${ct} | sort -u | wc -l` ] || exit 1",
```

Comment:
I stole this approach from: https://stackoverflow.com/questions/16349543/how-to-determine-if-the-content-of-one-file-is-included-in-the-content-of-anothe